src/controllers/comment.controller.js

```diff
@@ -0,0 +1,56 @@
+import commentService from '../services/comment.service';
+import response from '../helpers/response.helper';
+import Paginate from '../helpers/paginate.helper';
+
+
+/**
+ * This class will controll all action related to the
+ * comment
+ */
+class comment {
+  /**
+     * This method will help
+     * to comment to the trip request
+     * @param {Object} req request data
+     * @param {Object} res response data
+     * @returns { Object} return a user message
+     */
+  static async createTripRequestComment(req, res) {
```

Comment:
@shemaeric1234 Since the comment controller should be a generic controller, it would be better to name this function as createComment. Then you can pass relevant data. Kindly update.